cmd/operator-sdk/build/cmd.go

```diff
@@ -73,7 +74,10 @@ func createBuildCommand(imageBuilder, context, dockerFile, image string, imageBu
 
 	for _, bargs := range imageBuildArgs {
 		if bargs != "" {
-			splitArgs := strings.Fields(bargs)
+			splitArgs, err := shlex.Split(bargs)
+			if err != nil {
+				return nil, fmt.Errorf("image-build-args is not parseable (%v)", err)
+			}
 			args = append(args, splitArgs...)
```

Comment:
I think we have not a test for this option.
WDYT about add it [here](https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/subcommand.sh)?

Reply:
I like that idea a lot. Was wondering where a good place for a test would be.